os/net/ipv6/tcpip.c

```diff
@@ -121,6 +121,19 @@ uint8_t
 tcpip_output(const uip_lladdr_t *a)
 {
   int ret;
+
+  /* Tag Traffic Class if we are using TC for variable retrans */
+#if UIP_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS
+  if(uipbuf_get_attr(UIPBUF_ATTR_MAX_MAC_TRANSMISSIONS) !=
+     UIP_MAX_MAC_TRANSMISSIONS_UNDEFINED) {
```

Comment:
this seems to check something that is never set by default (only happens to be 0)

Reply:
uipopt.h?

Reply:
uipopt.h?

Reply:
I mean if it is intended as 0 by default, we might want to explicitly set it as `UIP_MAX_MAC_TRANSMISSIONS_UNDEFINED`, not just count on the fact that `UIP_MAX_MAC_TRANSMISSIONS_UNDEFINED` is 0 and leave the attribute uninitialized